pymongo/message.py

```diff
@@ -1007,33 +1009,83 @@ def unack_write(
         msg: bytes,
         max_doc_size: int,
         docs: list[Mapping[str, Any]],
+        client: MongoClient,
     ) -> Optional[Mapping[str, Any]]:
         """A proxy for Connection.unack_write that handles event publishing."""
+        command_logger = logging.getLogger("pymongo.command")
+        duration = datetime.datetime.now() - self.start_time
+        # TODO: add serverConnectionId
+        command_logger.debug(
+            StructuredMessage(
+                clientId=client._topology_settings._topology_id,
+                message="Command started",
+                command=cmd,
+                commandName=next(iter(cmd)),
+                databaseName=self.db_name,
+                requestId=request_id,
+                operationId=request_id,
+                driverConnectionId=self.conn.id,
+                serverHost=self.conn.address[0],
+                serverPort=self.conn.address[1],
+                serviceId=self.conn.service_id,
+            )
+        )
         if self.publish:
-            assert self.start_time is not None
-            duration = datetime.datetime.now() - self.start_time
             cmd = self._start(cmd, request_id, docs)
```

Comment:
The spec requires that `durationMS` include the time spent sending the message to the server and reading the reply. Since we only need to report `durationMS` in  `Command Succeeded` logs, I don't think we need this.